webui/react/src/shared/routes/history.ts

```diff
@@ -1,2 +1,2 @@
 import { createBrowserHistory } from 'history';
-export default createBrowserHistory({ basename: process.env.PUBLIC_URL });
+export default createBrowserHistory();
```

Comment:
**question:** this is ok to do? same question with the changes in `stripUrl`

Reply:
The latest `history` library doesnt provide `basename` anymore, and we need to update `history` as well as `react-router-dom`

Reply:
some use cases that rely on PUBLIC_URL are:
- serve ui from any base path: eg `/det` `/` and `/cat`

Reply:
I moved base name to [here](https://github.com/determined-ai/determined/pull/4619/files#diff-fe9fa55e9afd85c5adb046f05a63357a7ed70bca08fae9622936c71c5a6f39b6R23). I tested most of pages manually, but we should write tests before this PR is merged.